tests/test_full.nf.test.snap

```diff
@@ -0,0 +1,282 @@
+{
+    "antismash": {
+        "content": [
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true,
+            true
+        ],
+        "meta": {
+            "nf-test": "0.8.4",
+            "nextflow": "24.04.2"
+        },
+        "timestamp": "2024-07-02T20:18:53.947865807"
+    },
+    "abricate": {
+        "content": [
+            "ERZ1664501.txt:md5,1ace32d2d44194d635db31daa89a6fae",
+            "ERZ1664503.txt:md5,da890e2cef1b1a34ec035f6198f0a60b",
+            "ERZ1664504.txt:md5,3db1864a6efe5321379e89dcee34d505",
+            "ERZ1664505.txt:md5,317354f6091bad44ab5852399d48eb4a",
+            "ERZ1664506.txt:md5,8fcc568d6a15d1c3ac889169ce884093",
+            "ERZ1664507.txt:md5,0be8f76b35aca900f8f7fa6d2e7fc1f9",
+            "ERZ1664508.txt:md5,357da5e192d9a17b501446e181f41942",
+            "ERZ1664509.txt:md5,c14f4aef2c96e8c4f6688af35fe07a2c",
+            "ERZ1664510.txt:md5,0d88060e28b267a308271a2a2af38b12",
+            "ERZ1664511.txt:md5,46adf11d5d3952e3709ba05ec76b5e8a",
+            "ERZ1664515.txt:md5,4b7a0db47ac6e9baf723e6b2ef31bfc4",
+            "ERZ1664516.txt:md5,1ccfd94077fe329f7b30351aa846b327",
+            "ERZ1664517.txt:md5,8137ab84373a5300c9626a9459a2c935",
+            "ERZ1664518.txt:md5,db514f4bef8de0d4799f478e1807adc6",
+            "ERZ1664520.txt:md5,a3afa2368c941cdb0c4abd8efa855f0e",
+            "ERZ1664521.txt:md5,2849a78188c4793d4608ba1775da1d58",
+            "ERZ1664523.txt:md5,507e1710e7220965010ad8375b4c434a",
+            "ERZ1664524.txt:md5,b7d380fe3fbcb0fe2ac23823cb6b35e8",
+            "ERZ1664528.txt:md5,64aff1aaaab8b3d009edd40527446c08"
+        ],
+        "meta": {
+            "nf-test": "0.8.4",
+            "nextflow": "24.04.2"
+        },
+        "timestamp": "2024-07-02T20:18:29.527514564"
+    },
+    "fargene": {
+        "content": [
+            "results_summary.txt:md5,f69af9b0e18a0cfc934eb18b7e4bffe1",
```

Comment:
Everything the same? That seems strange when we have lots of samples files?

Reply:
It's indeed unfortunate but a valid result. There are no hits found for the beta-lactamases class 1_2. I'll update this file check to class a, where we have more (and different) hits and md5 sums

Reply:
Done in [33f55b5](https://github.com/nf-core/funcscan/pull/366/commits/33f55b5306573c630c959965bea94a71aef74375).

Reply:
Much better :)